fastlane/metadata/es-MX/name.txt

```diff
@@ -1 +0,0 @@
-WordPress - Constructor web
```

Comment:
My guess is this was deleted because of another case of `429 Too Many Requests` on `es-MX`, and `gp_downloadmetadata` treating any error without distinction as "no translation", hence this quite silent deletion during `fastlane finalize_release`.

See PR description for details.